src/unix/linux_like/linux/mod.rs

```diff
@@ -6169,3 +6290,88 @@ expand_align!();
 
 mod non_exhaustive;
 pub use self::non_exhaustive::*;
+
+mod ioctl {
+    const _IOC_NRBITS: u32 = 8;
+    const _IOC_TYPEBITS: u32 = 8;
```

Comment:
should this move somewhere else? 

I also get a style error on the const definition 

> src/unix/linux_like/linux/mod.rs:6295: constant found after module when it belongs before

not sure what that is about

Reply:
the error is back. Maybe the check is too strict, and just does not allow consts defined in inner modules? I can remove the module wrapper but bundling the symbols/logic with a `mod` is kind of nice